requirements.txt

```diff
@@ -29,3 +29,4 @@ aliyun-python-sdk-ecs==4.24.25
 arcaflow-plugin-sdk
 wheel
 git+https://github.com/vmware/vsphere-automation-sdk-python.git@v8.0.0.0
+git+https://github.com/arcalot/arcaflow-plugin-kill-pod.git@main
```

Comment:
Hitting "arcaflow-plugin-kill-pod 0.0.0 requires arcaflow-plugin-sdk>=0.8.0, but you'll have arcaflow-plugin-sdk 0.3.0 which is incompatible". We might want to add in a release tag for arcaflow-plugin-sdk. Thoughts?

Reply:
do we want to just add to the installation that we should always do 'pip install -U -r requirements' so that we continue to get the latest release of _arcaflow-plugin-sdk_?